tests/dflowfm/test_mdu.py

```diff
@@ -388,3 +392,115 @@ def test_loading_fmmodel_model_with_both_ini_and_xyn_obsfiles(self):
             assert actual_point.x == expected_point.x
             assert actual_point.y == expected_point.y
             assert actual_point.name == expected_point.name
+
+    def test_mdu_unknown_keywords_loading_gives_message_for_missing_keyword(
+        self, tmp_path, capsys
+    ):
+        tmp_mdu = """
+        [General]
+        fileVersion           = 1.09          
+        fileType              = modelDef      
+        program               = D-Flow FM     
+        version               = 1.2.100.66357 
+        autoStart             = 0             
+        pathsRelativeToParent = 0             
+        unknownkey            = something
+        """
+
+        tmp_mdu_path = tmp_path / "tmp.mdu"
+        tmp_mdu_path.write_text(tmp_mdu)
+
+        with patch(
+            "hydrolib.core.dflowfm.ini.models.INIBasedModel.Config"
+        ) as mock_config:
+            mock_config.extra = Extra.ignore
+
+            FMModel(filepath=tmp_mdu_path)
+            captured = capsys.readouterr()
+
+            section = "General"
+            name = "unknownkey"
+
+            assert (
+                f"Unknown keywords are detected in '{section}', these keywords will be dropped:"
+                in captured.out
+            )
+            assert name in captured.out
+
+    def test_mdu_unknown_keywords_loading_gives_message_for_missing_keyword2(
```

Comment:
Give it an appropriate name please

Reply:
tests werent updated yet it seems, will update them and rename them.